dags/burnham.py

```diff
@@ -202,35 +204,17 @@
 WANT_TEST_SPACE_SHIP_READY_PING = [{"count_documents": 3}]
 
 # Sensor templates for the different tables
-DISCOVERY_SENSOR_TEMPLATE = """
```

Comment:
If I recall correctly, we use query params in the queries we send to the burnham image and we pass in the query string base64-encoded.  But for the sensor query, Airflow is executing it directly and we don't have the base64 encoding issue, so we use string substitution.

It would be worth adding a comment here to further document that difference.

Reply:
Done! ✅